apps/mochi-web/components/MochiWidget/TokenPicker/MonikerItem.tsx

```diff
@@ -31,9 +31,7 @@ export const MonikerItem: React.FC<ItemProps> = ({ item, onSelect }) => (
         <span className="text-sm">{MonikerIcons.get(item.name)}</span>
       </div>
       <div className="flex flex-col flex-1 items-start">
-        <Heading as="h3" className="text-sm">
-          {item.name}
-        </Heading>
+        <Typography level="p5">{item.name}</Typography>
```

Comment:
Previously, it's been a `h3` tag but the change makes it a `p` tag. Can we keep the h3 tag?

Reply:
If the reason has been the styling, it's ok the use raw HTML heading tags instead of the Typography component. We can aim to ditch the Heading first.